src/Tribe/Attendees.php

```diff
@@ -575,7 +576,7 @@ private function generate_filtered_list( $event_id ) {
 		$filter_name = "manage_{$this->page_id}_columns";
 		add_filter( $filter_name, [ $this->attendees_table, 'get_columns' ], 15 );
 
-		$items = Tribe__Tickets__Tickets::get_event_attendees( $event_id );
+		$items = 'all' === $event_id ? Tickets::get_attendees_by_args()['attendees'] : Tickets::get_event_attendees( $event_id );
```

Comment:
The docblock for the filter above (and implicitly the whole method) indicates that `$event_id` should be an integer, but now we're passing a string. We should update the two docblocks to reflect the changed use.